holoviews/core/data/util.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+import sys
+
+import numpy as np
+
+def get_array_types():
+    array_types = (np.ndarray,)
+    if 'dask' in sys.modules:
+        import dask.array as da
+        array_types += (da.Array,)
+    return array_types
+
+def get_dask_array():
```

Comment:
I think `get_dask_array_module` or `dask_array_module` or even `da_module` or `get_da_module` would be clearer. You aren't getting the data type, you are getting a module...

Reply:
Sounds good.